### DISCUSSION TITLE
[Bug]: Prompt token count of 5677 exceeds max token count of 4095.

### DISCUSSION THREAD

**Discussion by SamWylde** [danny-avila/LibreChat#2404] [Troubleshooting]
### What happened?

Using any of the gpt-4 models, if the tokens exceed 4095, we get an error. Gpt-4 models should support 128,000 tokens.

"Something went wrong. Here's the specific error message we encountered: Prompt token count of 5677 exceeds max token count of 4095."

### Steps to Reproduce

1. Start a new conversation (gpt-4-turbo-preview is used for this example).
2. Use this prompt: https://docs.google.com/document/d/1tyRjbxhual4JIYUgL6kERDLwU4lfuF2BG56kIZ35Trc

### What browsers are you seeing the problem on?

Firefox

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/5250635/410a11ac-41ab-46e8-9cc4-64348487426e)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Try switching the “agent” model
![IMG_8044](https://github.com/danny-avila/LibreChat/assets/110412045/d7c5905f-78c3-4800-9f25-270741571c44)
![IMG_8045](https://github.com/danny-avila/LibreChat/assets/110412045/a813daec-edb3-4361-85da-9e15f57809a7)

**Reply by SamWylde** [reply]
Thank you for the help Danny! 

Unfortunately I still get the same error.

I also tried: gpt-4, gpt-4-0125-preview, and gpt-4-1106-preview.

![image](https://github.com/danny-avila/LibreChat/assets/5250635/d01db66c-5735-42d2-b64a-3f72ca5a2fac)

**Comment by danny-avila** [comment]
I’m not sure why the “Plugins” endpoint is giving you this issue but the “OpenAI” endpoint is working fine.

I will look into it, but switching the endpoint is a “quick fix”

<img width="414" alt="image" src="https://github.com/danny-avila/LibreChat/assets/110412045/450f8cf5-9bc6-4025-adf2-06b85873366f">

**Comment by abhranil26** [comment]
i am using groq api and using llama 3 70b which has max tokens limit of 9000 but librechat gives me this error 
2024-05-02 05:39:12 error: [handleAbortError] AI response error; aborting request: Prompt token count of 4104 exceeds max token count of 4095.
